lib/jurnal_api/response.rb

```diff
@@ -1,6 +1,8 @@
 module JurnalApi
   module Response
     def self.create( response_hash, ratelimit_hash )
+      return nil if response_hash.nil?
```

Comment:
when we send request delete, jurnal will return nil response

rescue in this file or in lib/jurnal_api/request.rb in `request` private method ?